app/src/main/java/com/yml/womensafety/repository/FirebaseRepository.kt

```diff
@@ -8,11 +8,13 @@ import com.yml.womensafety.FirebaseUtil
 import com.yml.womensafety.Response
 
 class FirebaseRepository {
+    private val databaseName = "name"
```

Comment:
This should be kept inside companion object and be made const

Reply:
Done